src/test/pybind/CMakeLists.txt

```diff
@@ -1,2 +1,4 @@
-add_ceph_test(test_ceph_daemon.py ${CMAKE_CURRENT_SOURCE_DIR}/test_ceph_daemon.py)
-add_ceph_test(test_ceph_argparse.py ${CMAKE_CURRENT_SOURCE_DIR}/test_ceph_argparse.py)
+if(NOT FREEBSD)
+  add_ceph_test(test_ceph_daemon.py ${CMAKE_CURRENT_SOURCE_DIR}/test_ceph_daemon.py)
+  add_ceph_test(test_ceph_argparse.py ${CMAKE_CURRENT_SOURCE_DIR}/test_ceph_argparse.py)
```

Comment:
@tchaikov 
After fixing the bang-path in the srcipt to be `/usr/local/bin/nosetests` running it just generates:
```
[~/Ceph/work.patch/ceph] wjw@freetest.digiware.nl> ./src/test/pybind/test_ceph_daemon.py
Unmatched ".
Exit 1
```
And for the time being I'd rather chase after other problems.